rsmtool/notebooks/header.ipynb

```diff
@@ -233,7 +234,11 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "Markdown('''This report presents the analysis for **{}**: {}'''.format(experiment_id, description))"
+    "display(Markdown('''This report presents the analysis for **{}**: {}'''.format(experiment_id, description)))\n",
+    "# print the excluded features in the report if exists.\n",
+    "if excluded_features:\n",
+    "    for feature in excluded_features:\n",
+    "        display(Markdown(f\"Feature **{feature}** was excluded from the model because its standard deviation in the training set is equal to 0.\"))"
```

Comment:
Couple of thoughts on this?

1. Is this the best place for this or should this go in one of the sections? I know that will be extra work so may be the is fine?
2. Instead of repeating the same sentence for each feature, can we just have something like: `The following features were excluded because their standard devision in the training set is equal to 0: F1, F2, F3`?